src/hotspot/share/code/codeCache.hpp

```diff
@@ -337,7 +337,7 @@ class CodeCache : AllStatic {
 // The relaxed iterators only hold the CodeCache_lock across next calls
 template <class T, class Filter, bool is_relaxed> class CodeBlobIterator : public StackObj {
  public:
-  enum LivenessFilter { all_blobs, only_not_unloading };
+  enum LivenessFilter { all, only_not_unloading };
```

Comment:
Thanks, I like this. FWIW, the `only` in `only_not_unloading` seems redundant. The code reads well without it, IMHO:

```
  // All nmethods
  NMethodIterator iter(NMethodIterator::all);
```
```
  // Those that are not unloading
  NMethodIterator iter(NMethodIterator::not_unloading);
```